packages/contracts/test/plugins/utils/addresslist.ts

```diff
@@ -49,54 +49,61 @@ describe('AddresslistMock', function () {
 
   context('addresslistLengthAtBlock', function () {
     it('returns the right length after addresses were added', async () => {
-      let tx1 = await addresslist.addAddresses([signers[0].address]);
+      const tx1 = await addresslist.addAddresses([signers[0].address]);
       await ethers.provider.send('evm_mine', []);
 
-      let tx2 = await addresslist.addAddresses([
+      const tx2 = await addresslist.addAddresses([
         signers[1].address,
         signers[2].address,
       ]);
       await ethers.provider.send('evm_mine', []);
 
+      const rc1 = await tx1.wait();
+      const rc2 = await tx2.wait();
```

Comment:
could be shortened to
```suggestion
      const [rc1, rc2] = await Promise.all([tx1.wait(), tx2.wait()]);
```